src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -337,11 +337,7 @@ array_rank_specifier
   ;
 
 function_pointer_type
-  : 'delegate' '*' syntax_token? '<' modified_type (',' modified_type)* '>'
-  ;
-
-modified_type
-  : modifier* type
+  : 'delegate' '*' syntax_token? '<' parameter (',' parameter)* '>'
```

Comment:
you ended up going with parameter?

Reply:
> you ended up going with parameter?

I did, in order to share parameter binding code.